go/private/rules/nogo.bzl

```diff
@@ -65,7 +65,7 @@ def _nogo_impl(ctx):
         go,
         struct(
             embed = [ctx.attr._nogo_srcs],
-            deps = analyzer_archives,
+            deps = analyzer_archives + [ctx.attr._go_difflib[GoArchive]],
```

Comment:
Adding this together with `ctx.attr.deps` in line 49 would be less surprising.

Reply:
We can't do that. `analyzer_archives` from Line 49 is used to populate `analyzer_importpaths` in Line 50. We don't want `_go_difflib` to be in `analyzer_importpaths`, because it cannot be imported as a nogo analyzer